cms/static/js/views/pages/container_subviews.js

```diff
@@ -523,7 +523,7 @@ function($, _, gettext, BaseView, ViewUtils, XBlockViewUtils, MoveXBlockUtils, H
                 tagValueElement.className = 'tagging-label-value';
 
                 tagContentElement.appendChild(tagValueElement);
-                parentElement.appendChild(tagContentElement);
+                parentElement?.appendChild(tagContentElement);
```

Comment:
[inform]: This change is temporary and will be removed after this PR https://github.com/openedx/edx-platform/pull/35828 is merged.

Reply:
#35828 has merged.  Will the conditional be removed?